include/crpropa/magneticField/TF17Field.h

```diff
@@ -0,0 +1,93 @@
+#ifndef CRPROPA_TF17FIELD_H
+#define CRPROPA_TF17FIELD_H
+
+#include "crpropa/magneticField/MagneticField.h"
+
+namespace crpropa {
+using namespace std;
+
+/**
+ Implements the TF2017 galactic magnetic field model, consisting of large-scale regular disk and halo fields.
```

Comment:
@tfitoussi: Could you add some doxygen descriptors like, e.g. [here](https://github.com/CRPropa/CRPropa3/blob/master/include/crpropa/magneticField/JF12FieldSolenoidal.h)?